web/packages/shared/components/Notification/Notification.tsx

```diff
@@ -175,13 +179,38 @@ function getRenderedContent(
           color="text.secondary"
           css={longerTextCss}
         >
+          {content.list && <ListRenderer list={content.list} />}
           {content.description}
         </Text>
+        {content.link && (
+          <Link href={content.link.href} target="_blank">
+            {content.link.text}
+          </Link>
+        )}
       </Flex>
     );
   }
 }
 
+function ListRenderer(props: { list: string[] }) {
```

Comment:
Changed to `List`. I increased the margin slightly, you could check if it helped?

Reply:
> Changed to `List`. I increased the margin slightly, you could check if it helped?

It didn't unfortunately but at least now I see that it probably has to do with how they're displayed in Connect rather the notifications themselves directly.

![list notifications in the storybook vs Connect](https://user-images.githubusercontent.com/27113/222399742-9cc4cf0e-8f1f-4cab-a44e-59efebe0e933.png)

Reply:
This is strange, I don't see what could cause it.

Reply:
Do you have a device on which you could check this yourself or should I try to debug this?

Reply:
I don't have a display where I could reproduce it, so I would be grateful if you could take a look at it!

Reply:
This is because by default `padding-inline-start` sets the indentation of the actual element inside `<li>` and the bullet point is not considered in this calculation.

Setting `list-style-position: inside` includes the bullet point in the calculation and thus makes sure that it is never cut off. I'll submit a patch in a sec.

Reply:
And the difference between the storybook and the app comes from the fact that I opened the storybook in Firefox which gives different styles to the `::marker` element.

Reply:
OTOH `list-style-position: inside` looks bad if the list item spans multiple lines.

It's impossible to align the bullet point to the left with `padding-inline-start` on both full HD and Retina without media queries so I'm just going to use 2em for both.

Reply:
>OTOH list-style-position: inside looks bad if the list item spans multiple lines.

I see :(

So you just want to change `padding-inline-start` from 13px to 2em, correct?